server/appconfig/welcome.go

```diff
@@ -1,7 +1,16 @@
 package appconfig
 
-import "github.com/jitsucom/jitsu/server/logging"
+import "fmt"
+
+const banner = "========================================================================\n\n" +
+	"Welcome to EventNative %s!\n\n" +
+	"EventNative is a data collection platform developed by Jitsu\n\n" +
+	" 📚 Documentation: https://docs.eventnative.org/\n" +
+	" 🌎 Website: https://jitsu.com\n" +
+	" 💪 Follow us on twitter: https://twitter.com/jitsucom\n" +
+	" 💬 Join our Slack: https://jitsu.com/slack\n\n" +
+	"========================================================================\n"
 
 func logWelcomeBanner(version string) {
-	logging.Infof("\nWelcome to EventNative %s developed by Jitsu (https://jitsu.com)\n  * Documentation: https://docs.eventnative.org/\n", version)
+	fmt.Printf(banner, version)
```

Comment:
Please use the `logging` pkg function for writing Welcome Banner into stdout as well as into the log file.